src/components/Icon/Icon.scss

```diff
@@ -1,23 +1,26 @@
 @import '../../index.scss';
-
 @font-face {
-	font-family: 'icomoon';
-	src:url('/fonts/icomoon.eot');
-	src:url('/fonts/icomoon.eot?#iefix') format('embedded-opentype'),
-		url('/fonts/icomoon.woff') format('woff'),
-		url('/fonts/icomoon.ttf') format('truetype'),
-		url('/fonts/icomoon.svg?#icomoon') format('svg');
-	font-weight: normal;
-	font-style: normal;
+    font-family: 'icomoon';
+    src: url('/fonts/icomoon.eot');
+    src: url('/fonts/icomoon.eot?#iefix') format('embedded-opentype'), url('/fonts/icomoon.woff') format('woff'), url('/fonts/icomoon.ttf') format('truetype'), url('/fonts/icomoon.svg?#icomoon') format('svg');
+    font-weight: normal;
+    font-style: normal;
 }
 
 .icon {
     -moz-osx-font-smoothing: grayscale;
-	-webkit-font-smoothing: antialiased;
-	display: inline-block;
+    -webkit-font-smoothing: antialiased;
+    display: inline-block;
     font-family: icomoon;
     font-style: normal;
-	margin-right: 5px;
+    margin-right: 5px;
+}
+
+.svg {
```

Comment:
maybe nest the svg class inside the icon class to limit the scope to the icon component.

Reply:
Ok, but currently in the .icon class is many font-related properties which are not necessary for svgs.

Reply:
indeed, you are correct, i mistakenly thought that there was some parent element, but it always returned <i/> directoly. how about removing the svg class and the using the following selectors
i.icon, svg.icon.

in the icon component we would the return either return 
```<i { ...getNativeAttributes(props, htmlElementAttributes) } className={iconClassName} />;```
or
```<svg className={iconClassName} />;```


with this the iconClassName will work as before and the component will respect the props.className.